internal/mid/errors.go

```diff
@@ -57,6 +60,12 @@ func ErrorHandler(before web.Handler) web.Handler {
 			// Respond with the error.
 			web.Error(ctx, log, w, err)
```

Comment:
Do we care that the shutdown error will display the message `web value missing from context` to the end user?

Reply:
I was thinking about that. We should